content/blogs/newsletter-104.md

```diff
@@ -0,0 +1,124 @@
+Their main focus is to maintain the Shell and Audio components, as well as on any other module that requires assistance.
+
+## Symmetric Multiprocessing (SMP) support status & GCC 13 preparations
+Symmetric multiprocessing support for the AMD64 architecture is being extensively improved thanks to Timo Kreuzer (tkreuzer).
+In particular, he has implemented support for processor freezing and switching ([PR #6581](https://github.com/reactos/reactos/pull/6581)) so that every logical processor is frozen or switched properly.
+In addition, he implemented basic support for IPIs (inter-processor interrupts) which are needed for SMP scheduling, debugging, and more ([PR #6130](https://github.com/reactos/reactos/pull/6130)).
+These are only limited to APC (Asynchronous Procedure Calls), DPC (Deferred Procedure Calls) and freeze IPIs for the time being.
+
+Another important improvement for SMP is the kernel initialization in the AMD64 build architecture, which was revamped by Timo ([PR #6110](https://github.com/reactos/reactos/pull/6110)).
+This allows SMP to work properly.
+
+{{< gallery >}}
+{{< figure link="/img/blogs/newsletter-104/blender_smp.png" src="/img/blogs/newsletter-104/blender_smp.png" caption="Blender running in ReactOS 64-bit with 2 cores">}}
+{{< /gallery >}}
+
+Timo also helped fix some bugs to be able to compile ReactOS in the GCC 13 build environment ([PR #6824](https://github.com/reactos/reactos/pull/6824)).
+Much of the thanks for preparing ReactOS to build on GCC 13 goes to Simone Mario Lombardo (simonelombardo) for making ReactOS bootable.
+
+In the meantime, Justin Miller (The_DarkFire) is focusing on the x86 architecture of the SMP area.
+He worked on setting up application processor bootup ([PR #5879](https://github.com/reactos/reactos/pull/5879)) in x86 SMP HAL and is planning to further improve x86 SMP.
+On top of all of this experiments have begun to allow ReactOS to make use of the new SMP aware HAL.
+
+{{< gallery >}}
+{{< figure link="/img/blogs/newsletter-104/smp_x86.png" src="/img/blogs/newsletter-104/smp_x86.png" caption="Experimenting the 32-bit SMP build of ReactOS">}}
+{{< figure link="/img/blogs/newsletter-104/smp_cpuz.png" src="/img/blogs/newsletter-104/smp_cpuz.png" caption="ReactOS SMP 32-bit showing the benchmark results with CPU-Z">}}
+{{< /gallery >}}
+
+**It is worth noting that SMP is still heavily under development and there is no exact estimate of when the multi-core support will finally be ready, so stay tuned for more updates!**
+
+## User-mode WINE modules syncing
+Timo, Justin and several other contributors are making efforts to bring up-to-date some of the user-mode modules that we borrow from WINE.
+As you may know; ReactOS shares many components in user mode with WINE, although our shell is in many cases an exception to this the bulk of the usermode DLLs that provide APIs to applications are synced with the WINE project as base.
```

Comment:
Suggestion:
As you may know, ReactOS shares many components in user mode with WINE, although our shell is in many cases an exception to this, the bulk of the usermode DLLs that provide APIs to applications are synced with the WINE project as their base.